stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,95 +37,101 @@ class Worksheet(_WorkbookChild):
     PAPERSIZE_A5: str
     ORIENTATION_PORTRAIT: str
     ORIENTATION_LANDSCAPE: str
-    def __init__(self, parent, title: Incomplete | None = ...) -> None: ...
+    def __init__(self, parent: Workbook | None, title: str | _Decodable | None = ...) -> None: ...
     @property
-    def sheet_view(self): ...
+    def sheet_view(self) -> SheetView: ...
     @property
-    def selected_cell(self): ...
+    def selected_cell(self) -> str: ...
     @property
-    def active_cell(self): ...
+    def active_cell(self) -> str: ...
```

Comment:
Conflict: As seen by `Selection.__init__`, these are the string representation of cells. Descriptors also force these for be strings.

Reply:
Good point, I agree -- these should both be strings.